docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -25,10 +25,17 @@ the Application Service and Database Service.
 
 To diagnose problems, you can configure the `teleport` process to run with
 verbose logging enabled by passing it the `-d` flag. `teleport` will write logs
-to stderr.
+to stderr. 
 
-Logs will resemble the following (these logs were printed while joining a Node
-to a cluster, then terminating the `teleport` process on the Node):
+Alternatively to the `-d` flag you can set the log level within the Teleport configuration file.
+```yaml
+teleport:
+  log:
+    severity: DEBUG
+```
+
+Restart the `teleport` process to apply the modified log level. Logs will resemble the following
+(these logs were printed while joining a Node to a cluster, then terminating the `teleport` process on the Node):
```

Comment:
Suggestions for formatting, brevity, and clarity:

````suggestion
Alternatively, you can set the log level from the Teleport configuration file:

```yaml
teleport:
  log:
    severity: DEBUG
```

Restart the `teleport` process to apply the modified log level. Logs will resemble
the following (these logs were printed while joining a Node to a cluster, then
terminating the `teleport` process on the Node):
````